.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   test_node_10:
```

Comment:
Could change the name of that task (and the job at the bottom) as well.

Reply:
nice catch, done.